extensions/lifecycle/lifecycleConsumer/task.js

```diff
@@ -0,0 +1,18 @@
+'use strict'; // eslint-disable-line
+const werelogs = require('werelogs');
+
+const LifecycleConsumer = require('./LifecycleConsumer');
+
+const config = require('../../../conf/Config');
+const zkConfig = config.zookeeper;
+const lcConfig = config.extensions.lifecycle;
+const s3Config = config.s3;
+const authConfig = config.auth;
+
+const lifecycleConsumer = new LifecycleConsumer(
+    zkConfig, lcConfig, s3Config, authConfig);
+
+werelogs.configure({ level: config.log.logLevel,
```

Comment:
I noticed this pattern also in the queue processor task file. I may be mistaken but I don't think configuring `werelogs` like this without using the logger instance does anything.

Reply:
This has been changed recently (a few months ago still) in werelogs, this is now the common way of setting the log level globally for the process.